packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx

```diff
@@ -79,6 +79,10 @@ export interface EntryCardPropTypes extends BaseCardProps {
    * Changes the height of the component. When small will also ensure thumbnail and description aren't rendered
    */
   size?: EntryCardSize;
+  /**
+   * Render custom body for the entry. This will override the props title, description and thumbnailElement
+   */
+  customBody?: React.ReactNode;
```

Comment:
Looks like `customBody` acts like `children` here. Maybe it would be better to name it `children`?

Reply:
Maybe? Naming it `children` maybe removes some semantic knowledge? I.e. if it is named `children` will someone think that it replaces everything inside the entry card? Similarly to how it works in a plain card?